modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -866,6 +914,33 @@ GrainTracker::trackGrains()
   }
 }
 
+void 
+GrainTracker::createAdjacentIDVector() // by weipeng
```

Comment:
This is good but might be good to move to your new class. Again, most of the base class usage doesn't require this and this might start to be a time sync for systems with very large amounts of grains since the "haloIntersect" method can be expensive.

Reply:
OK， I took your advice and moved createAdjacentIDVector into GrainTrackerMerge.